### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by digitaldesigndj**
I've added some information and an a link to the README file.

inbetween is not a word (now I'm sweating, cause I might not have spelled everything correctly...)

This game is _awesome_. Thanks for the great times! Can't wait for 0.17!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 # factorio-data
-Tracks changes of the lua prototype definitions in factorio inbetween releases.
+
+This repo has been made publicly available to help mod authors. :smiley::+1:
+
+It's purpose is to allow the public to track changes to the Factorio **lua prototype definitions**.
```
Comment (dgw) [inline README.md]: `Its purpose`*

Also, "Lua" is capitalized.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
 
 See https://github.com/wube/factorio-data/releases for past versions.
 Compare versions using git on the command line, or by using magic github comparison urls, 
 eg: https://github.com/wube/factorio-data/compare/0.14.23...0.15.40
+
+## Some Potential Use Cases of Data
+
+* Online references, cheat sheets, and calculators; can use it to automatically update when new versions are released.
```
Comment (dgw) [inline README.md]: The semicolon can go, imo.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
 # factorio-data
-Tracks changes of the lua prototype definitions in factorio inbetween releases.
+
```
[the remark this answers is not in the document]
Comment (digitaldesigndj) [reply README.md]: That was part of the original. I was focused on adding some content from the FFF post. But if you like the MOD authors line more 😺

[the remark this answers is not in the document]
Comment (sparr) [reply README.md]: This PR removes that line from the original. I don't really care what is on that line, aside from it being some sort of description, I just don't want the second line of the file to be blank.

```
# repo-name
one-line description
[blank line]
more text
goes here
as much as
you want
```

**Comment by wheybags** [comment]
"merged" manually, with some edits: https://github.com/wube/factorio-data/commit/70437ace915c64dc0333553494f0af9b0ec2349d

**Pull request closed without merging**